packages/serverpod/lib/src/server/server.dart

```diff
@@ -337,7 +338,7 @@ class Server {
 
   Future<Result> _handleUriCall(
       Uri uri, String body, HttpRequest request) async {
-    var path = uri.path.substring(1);
+    var path = uri.path.subString(1);
```

Comment:
Suggestion(Open for discussion): If a slash path is not provided, pass on the string without formatting to handleUriCalls.

It seems like the only purpose of this line is to strip any leading slash. In the case that we don't receive a path starting with a slash, doesn't it make sense to parse it as a complete path and pass it onto the handleUriCall call method and then ensure that it can handle any path provided? 

Inside of handleUriCall it seems like we handle the case that the path would be empty or not recognized and return a `ResultInvalidParams` object.

Reply:
This has been updated as of the requirement.
<img width="1261" alt="Screenshot 2024-01-17 at 3 34 46 PM" src="https://github.com/serverpod/serverpod/assets/35523357/410ff8d1-ce87-44eb-90a7-0c4ada7fb347">